dist/tsconfig.tsbuildinfo

```diff
@@ -1 +1 @@
],[35,2],[36,2],[37,2],[38,2],[43,3],[40,2],[41,2],[42,2],[50,4],[45,4],[51,5],[49,4],[48,4],[46,4],[47,4],[52,6],[53,7],[54,8],[56,7],[55,8],[66,9],[57,7],[58,7],[63,7],[59,8],[60,8],[61,8],[62,7],[64,10],[24,11],[67,12],[68,13],[71,12],[70,14],[69,14],[80,15],[72,12],[73,12],[77,12],[74,14],[75,13],[76,12],[78,16],[27,17],[123,18],[81,19],[82,19],[83,19],[84,20],[85,20],[86,20],[87,19],[88,20],[90,20],[89,19],[98,21],[91,20],[92,20],[95,20],[93,19],[94,20],[96,22],[28,11],[99,23],[100,23],[101,24],[103,25],[102,24],[29,26],[104,27],[105,28],[106,28],[108,29],[122,30],[109,27],[110,27],[111,27],[112,27],[113,28],[115,31],[116,27],[117,32],[118,28],[119,27],[120,33],[26,34],[30,35],[380,36],[285,37],[126,38],[127,39],[128,38],[129,38],[130,39],[131,39],[132,38],[133,39],[134,38],[141,40],[135,39],[136,39],[137,38],[138,38],[139,41],[140,38],[143,42],[144,43],[145,42],[146,42],[147,43],[148,43],[149,42],[150,44],[151,43],[152,43],[158,45],[153,46],[154,43],[155,42],[156,42],[157,47],[142,48],[159,49],[160,49],[161,49],[162,49],[163,49],[164,50],[165,50],[166,49],[167,50],[168,49],[169,49],[170,49],[171,49],[172,49],[185,51],[173,49],[174,49],[175,52],[176,49],[177,50],[178,49],[179,49],[180,49],[181,50],[182,49],[183,49],[184,50],[186,53],[187,53],[188,53],[189,53],[190,53],[193,54],[191,54],[194,54],[195,54],[192,53],[200,55],[197,54],[196,54],[198,53],[199,52],[354,56],[201,48],[202,26],[207,57],[203,26],[204,26],[205,26],[208,26],[209,26],[211,58],[212,26],[210,26],[213,26],[214,26],[215,26],[216,59],[217,26],[218,26],[219,26],[220,26],[221,26],[222,26],[223,26],[224,26],[241,60],[225,26],[226,26],[227,26],[228,26],[229,26],[230,26],[231,26],[232,26],[233,26],[234,26],[235,26],[236,26],[237,26],[238,26],[239,26],[240,61],[242,48],[243,48],[244,48],[245,48],[246,48],[247,48],[248,48],[249,48],[250,48],[251,48],[252,48],[253,48],[254,48],[255,59],[274,62],[256,48],[257,48],[258,48],[259,48],[260,48],[261,48],[262,48],[263,48],[264,48],[265,48],[266,48],[267,48],[268,48],[269,48],[270,48],[271,48],[272,48],[273,50],[275,63],[276,63],[277,63],[278,63],[279,63],[280,63],[281,63],[284,64],[282,63],[283,65],[299,66],[286,67],[288,68],[289,67],[290,67],[291,67],[292,68],[293,67],[294,67],[295,67],[296,67],[297,67],[298,67],[287,48],[337,69],[333,67],[336,67],[334,67],[335,67],[339,70],[341,71],[340,67],[343,72],[347,73],[344,67],[345,70],[346,73],[364,74],[348,75],[349,76],[350,77],[351,77],[352,77],[355,78],[356,79],[363,80],[357,77],[358,77],[359,77],[360,81],[361,77],[362,82],[342,48],[365,67],[366,67],[367,83],[368,67],[371,84],[370,73],[369,67],[372,67],[373,85],[379,86],[374,87],[375,87],[376,87],[377,88],[378,87],[300,19],[301,8],[302,14],[303,14],[304,14],[305,14],[306,8],[307,14],[308,12],[321,89],[309,19],[310,8],[311,90],[312,8],[313,14],[314,14],[315,8],[316,14],[317,8],[318,8],[319,12],[320,7],[324,91],[322,26],[323,26],[325,73],[327,92],[332,93],[389,94],[390,95],[388,96],[387,97],[384,98],[383,99],[382,100],[391,101]]},"version":"5.5.3"}
```

Comment:
Does this need to be checked in? Seems like tsconfig.tsbuildinfo is mostly a cache file. Should be removed and added to gitignore?